wikimedia/wikipedia-ios
# Image Recommendations - Prevent view from rotating on iPhone

**Phabricator:** https://phabricator.wikimedia.org/T360037

### Notes
* Prevents Image recs view from rotating on iPhone
* Rotating on iPad is still possible in all directions
* The orientation is locked on the ImageRecommendationsViewController. Since this VC is embedded in a Navigation Controller, this PR overrides some properties in our Root Nav Controller to allow for each child VC to control rotation

### Test Steps
1. Run the app on an iPhone, go to image recommendations
2. Make sure the device rotation is unlocked
3. Rotate in all directions
4. The view should not rotate
5. test other features - they should rotate normally
6. test on iPad - the Image Recs view should be able to rotate
